docs/samples/kafka_producer.py

```diff
@@ -0,0 +1,89 @@
+from feathr import AvroJsonSchema, KafKaSource, KafkaConfig
+import os
```

Comment:
maybe update the feathr\feathr_project\test\prep_azure_kafka_test_data.py instead of adding new ones

Reply:
Removed this file.